lib/active_merchant/billing/gateways/elavon.rb

```diff
@@ -370,6 +372,15 @@ def merchant_initiated_unscheduled(options)
         return 'Y' if options.dig(:stored_credential, :initiator) == 'merchant' && options.dig(:stored_credential, :reason_type) == 'unscheduled'
       end
 
+      def recurring_transaction(options)
+        if options.dig(:stored_credential, :reason_type) == 'recurring' && options.dig(:store_credential, :initial_transaction) == true
+          return options[:ssl_get_token] if options[:ssl_get_token] && options[:ssl_add_token] if options[:ssl_add_token] && options[:merchant_initiated_unscheduled] if options[:merchant_initiated_unscheduled]
```

Comment:
So, my question for these is, would they be any other value other than 'Y'?

Reply:
And do we still need to check for this field, `options.dig(:stored_credential, :initiator) == 'merchant'`?